homeassistant/components/eheimdigital/quality_scale.yaml

```diff
@@ -22,3 +22,49 @@ rules:
   test-before-configure: done
   test-before-setup: done
   unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: No service actions implemented.
```

Comment:
Light is a mutable entity, so we maybe need error handling there

Reply:
Exception handling is added.